deployment/helm/Chart.lock

```diff
@@ -5,5 +5,5 @@ dependencies:
 - name: redis
   repository: https://kubernetes-charts.storage.googleapis.com
   version: 9.4.3
-digest: sha256:63b5c2d6fa6dbceb477ec60d94c9d4f3f14e0ee85738b32880213282ab9a7f0e
```

Comment:
Did anything actually change here?

Reply:
I had out of sync problem locally with the previous digest. I had to remove the lock file, move the dependencies into the Chart.yml and finally build it again, that produces the new digest. Effectively there should be no changes.